ui/components/ui/account-list/account-list.js

```diff
@@ -26,16 +43,19 @@ const AccountList = ({
   const selectedAccountScrollRef = useRef(null);
   useLayoutEffect(() => {
     selectedAccountScrollRef.current?.scrollIntoView({ behavior: 'smooth' });
-  }, [selectedAccounts]);
+  }, []);
+
+  const [firstSelectedAccount] = selectedAccounts;
 
   const Header = () => {
-    let checked;
+    let checked = false;
+    let isIndeterminate = false;
     if (allAreSelected()) {
-      checked = CHECKED;
+      checked = true;
     } else if (selectedAccounts.size === 0) {
-      checked = UNCHECKED;
+      checked = false;
     } else {
-      checked = INDETERMINATE;
+      isIndeterminate = true;
```

Comment:
Why change these values from constants to explicit booleans?

Reply:
These were constants from the old deprecated Checkbox component. I replaced everything with new Checkbox and followed its ways of configurations and usage.